content/5_Plat/Sweep_Line.mdx

```diff
@@ -148,11 +191,13 @@ int main() {
 	}
 
 	int ans = 0, ptr = 0;
+	Event curr_post = {0, 0, {origin.x, n}};
 	for (Event i : events) {
-		while (ptr != 4 * n && walls[ptr] < i) {
+		while (ptr != 4 * n && curr_post < i) {
```

Comment:
the runtime is still proportional to n

Reply:
Could the pr still be merged without the optimized solution for now? We could open a GitHub issue for it and do it later

Reply:
alright